symtab_builder.cc

```diff
@@ -42,6 +44,13 @@ uintptr_t SymtabBuilder::AddSym(const Syminfo& sym) {
     return index;
 }
 
+// Returns true when the symbol specified with (name, soname, version) is
+// defined.
+// TODO(akawashiro) Is it true? Is there any defined symbols whose st_value is
+// 0?
+// When the specified symbol is not defined, SymtabBuilder::Resolve pushes it
+// to sym_ and exposed_syms_.
```

Comment:
How about

and fills the index of the added symbol to `val_or_index`

?

Reply:
I see.